python/analyzers/treeVariables.py

```diff
@@ -20,6 +20,7 @@ def default():
 # event variables
 event_vars = [
     Variable('run', type=int),
+    Variable('lumi', lambda ev : ev.lumi, type=int),
     Variable('event', lambda ev : ev.eventId, type=int),
```

Comment:
`event` branch needs to become `long int`, otherwise we'd get negative event numbers

Reply:
@dehuazhu this is important (actually shame on me for not fixing this earlier)